### PR TITLE
[CHI-157] Add javascript range slider component

### PR DESCRIPTION
**Pull request by dani-cl-madrid**
Uncompleted feature. 
JavaScript-dependant consistent design for the range slider for WebKit browsers. 
Two approaches for the new range slider design with labels below it (please, check this @mattnickles )

### CODE REVIEW

File: src/chi/components/range-slider/range-slider.scss [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-  border-radius: 12px;
+  border-radius: 0.75rem;
   box-sizing: border-box;
   cursor: pointer;
   display: block;
-  height: 20px;
-  transition: all 0.2s ease-in-out;
-  width: 20px;
+  height: 1.25rem;
+  //transition: all 0.2s ease-in-out;
```
Comment (jllr) [inline src/chi/components/range-slider/range-slider.scss]: @dani-cl-madrid comments must be removed

File: src/chi/components/range-slider/range-slider.scss [hunk 2]
```diff
@@ -23,10 +23,10 @@
 
 @mixin track-style {
   background-color: set-color(grey, 30);
-  border-radius: 10px;
+  border-radius: 0.25rem;
   box-sizing: border-box;
-  height: 8px;
-  transition: all 0.2s ease-in-out;
+  height: 0.5rem;
+  //transition: all 0.2s ease-in-out;
```
Comment (jllr) [inline src/chi/components/range-slider/range-slider.scss]: @dani-cl-madrid comments must be removed

File: src/website/views/components/forms/range-slider.pug [hunk 3]
```diff
@@ -37,3 +42,239 @@ h4 With Labels
...
+
+
+h3 Radio buttons as range slider
+
+p.-text
+  | Range slider can easily work in the way radio buttons do. Chi has a special design for this purpose.
+
+.example.-mb--3
+  .-p--3
+    .m-formGroups
```
Comment (mattnickles) [inline src/website/views/components/forms/range-slider.pug]: Should be singular `m-formGroup`

File: src/website/views/components/forms/range-slider.pug [hunk 4]
```diff
@@ -37,3 +42,239 @@ h4 With Labels
...
+      <div class="a-inputWrapper -w--100">
+        <input class="a-input" type="range">
+        <div class="a-input__progress"></div>
+        <div class="a-input__thumb"></div>
+      </div>
+      <label class="-text -ml--2">5TB</label>
+    </div>
+
+
+h3 Radio buttons as range slider
```
Comment (mattnickles) [inline src/website/views/components/forms/range-slider.pug]: Can we present the JS examples with Javascript? I understand why they're not but to the first time viewer, these examples look broken

**Pull request merged**